app/src/main/res/layout/fragment_home.xml

```diff
@@ -1,12 +1,55 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
-    <TextView
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:gravity="center"
+    android:orientation="vertical"
+    android:padding="1dp"
+    android:clipChildren="false">
+
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/home_title_height"
+        android:orientation="horizontal"
+        android:layout_marginTop="@dimen/home_title_marginTop"
+        android:clipChildren="false">
+
+        <ImageView
+            android:layout_width="@dimen/home_wolox_logo_width"
+            android:layout_height="wrap_content"
+            android:src="@drawable/wolox_logo" />
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textSize="@dimen/home_wolox_name_textSize"
+            android:text="@string/home_wolox"/>
+    </LinearLayout>
+
+
+    <android.support.design.widget.TabLayout
+        android:id="@+id/home_tab_layout"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:text="HOME"/>
+        android:layout_height="@dimen/home_tab_layout_height"/>
+
+
+    <android.support.v4.view.ViewPager
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/home_pager"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
+
+    <android.support.v7.widget.RecyclerView
+        android:id="@+id/home_recycler_view"
+        android:scrollbars="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"/>
+
 
-</LinearLayout>
+    <!--<View-->
+        <!--android:layout_width="match_parent"-->
+        <!--android:layout_height="0dp"-->
+        <!--android:layout_weight="1"/>-->
 
+</LinearLayout>
```

Comment:
Remember to not to have more than 2 empty lines :smile: I comment it here to no mention it in all the file :ok_hand: